src/styles/SearchBar.css

```
@@ -0,0 +1,28 @@
+.input-search-bar {
+    width: 668px;
+    height: 36px;
+    border-radius: 99px;
+    border: none;
+    background: url(../utils/images/lupa.svg) 15px no-repeat #EDEDF0;
+    padding-left: 45px;
+}
+
+.button-search-bar {
+    width: 88px;
+    height: 36px;
+    border-radius: 99px;
+    background: #FFFFFF;
```

Acho que o ESlint reclama um pouco da declaração de cores usando # ou neste caso, ele sugere, para as cores primárias, o uso do nome da cor 😄 . O que acha de trocar para `white`?
```suggestion
    background: white;
```

---

Descobri isso agora kkk